ember.md

```diff
@@ -357,3 +359,20 @@ Create a build definition for the app that will:
 
 ### 11. Checklist Violations
 `npm test` (in the build definition) will catch errors and reject build
+
+
+
+<a name="addons"></a>
+## Addons
+
+<a name="versioning"></a>
```

Comment:
I'm not sure if this is needed since the heading has numbers in it. @ASH-Bryan do you know?